packages/angular/src/utils/versions.ts

```diff
@@ -1,7 +1,7 @@
 export const nxVersion = '*';
-export const angularVersion = '^8.0.0';
-export const angularDevkitVersion = '^0.800.1';
+export const angularVersion = '^8.2.0';
+export const angularDevkitVersion = '^0.803.2';
 export const angularJsVersion = '1.6.6';
-export const ngrxVersion = '8.1.0';
+export const ngrxVersion = '8.3.0';
 export const rxjsVersion = '~6.4.0';
```

Comment:
@brandonroberts rxjs 6.5.3 is out, any reason to not get that one synced?
and thanks for this work!
P/S. I wonder if there will be a day to sync bugfix releases, as 8.0.3 and 0.803.3 is out already